src/render/md.jl

```diff
@@ -1,17 +1,43 @@
 ## Docs-specific rendering ––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––––
 
-function writemime(io::IO, mime::MIME"text/md", docs::Docs{:md})
+function writemime(io::IO, mime::MIME"text/md", docs::Docs{:md}, mdstyle::Dict{ASCIIString, ASCIIString} = DEFAULT_MDSTYLE)
     println(io, docs.data)
 end
 
 ## General markdown rendering ------------------------–––––––––––––––––––––––––––––––––––
 
-function save(file::String, mime::MIME"text/md", doc::Metadata; mathjax = false, include_internal = true)
+const DEFAULT_MDSTYLE = Dict{ASCIIString, ASCIIString}([
+  ("header"         , "#"),
+  ("objname"        , "####"),
+  ("meta"           , "**"),
+  ("exported"       , "##"),
+  ("internal"       , "##"),
+])
+
+const HTAGS = ["#", "##", "###", "####", "#####", "######"]
+const STYLETAGS = ["", "*", "**"]
+
+print_help(io::IO, cv::ASCIIString, item) = cv in HTAGS ? println(io, "$cv $item") : println(io, cv, item, cv)
+
+function validate(mdstyle::Dict{ASCIIString, ASCIIString})
+    reg_keys = ["header", "objname", "meta", "exported", "internal"]
```

Comment:
So we don't have to keep 2 lists of the keys in sync, perhaps use `reg_keys = keys(DEFAULT_MDSTYLE)` here.